docs/pages/versions/unversioned/sdk/router-split-view.mdx

```diff
@@ -0,0 +1,395 @@
+---
+title: Router Split View
+description: An Expo Router submodule that provides native split view layout.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-router'
+packageName: 'expo-router'
+platforms: ['ios']
+isAlpha: true
+---
+
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import APISection from '~/components/plugins/APISection';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { ConfigPluginExample } from '~/ui/components/ConfigSection';
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { FileTree } from '~/ui/components/FileTree';
+
+> **important** SplitView is currently in [alpha](/more/release-statuses/#alpha) and available in [canary releases](/versions/latest/#using-pre-release-versions) of `expo-router`. It requires configuration in **app.json** and is not available in Expo Go.
+
+`expo-router/unstable-split-view` is a submodule of `expo-router` and exports components to build split view layouts using [platform-native system split views](https://developer.apple.com/design/human-interface-guidelines/split-views).
+
+> See the [Expo Router](./router) reference for more information about the file-based routing library for native and web app.
+
+## Installation
+
+To use `expo-router/unstable-split-view` in your project, you need to install `expo-router` in your project. Follow the instructions from Expo Router's installation guide:
+
+<BoxLink
+  title="Install Expo Router"
+  description="Learn how to install Expo Router in your project."
+  href="/router/installation/"
+  Icon={BookOpen02Icon}
+/>
+
+## Configuration in app config
+
+To enable Split View support, add the `unstable_splitView` property to the `expo-router` plugin configuration in your app config file (app.json or app.config.js):
```

Comment:
```suggestion
To enable Split View support, add the `unstable_splitView` property to the `expo-router` plugin configuration in your [app config](/workflow/configuration/) file:
```

Let's link the app config doc here since `app.config.ts` is also a valid format and adding all formats is too verbose as per our writing style guide.